RetailCoder.VBE/Refactorings/EncapsulateField/PropertyGenerator.cs

```diff
@@ -11,9 +11,10 @@ public class PropertyGenerator
         public bool GenerateLetter { get; set; }
         public bool GenerateSetter { get; set; }
 
-        public string AllPropertyCode => GetterCode +
-                                         (GenerateLetter ? LetterCode : string.Empty) +
-                                         (GenerateSetter ? SetterCode : string.Empty);
+        public string AllPropertyCode => string.Format("{0}{1}{2}",
+                                                        GetterCode,
+                                                        (GenerateLetter ? LetterCode : string.Empty),
+                                                        (GenerateSetter ? SetterCode : string.Empty));
```

Comment:
Could just use an interpolated string instead, right?

Reply:
It may just be me, but `$"{GetterCode}{(GenerateLetter ? LetterCode : string.Empty)}{(GenerateSetter ? SetterCode : string.Empty)}"` feels crowded. It seemed clearer in the current format.